pkg/sysinfo/sysinfo.go

```diff
@@ -50,17 +47,6 @@ func New(quiet bool) *SysInfo {
 		}
 	}
 
-	// Checek if ipv4_forward is disabled.
-	if data, err := ioutil.ReadFile("/proc/sys/net/ipv4/ip_forward"); os.IsNotExist(err) {
```

Comment:
But `ip_forward` is critical for some docker functions. Swap memory breaks nothing in default docker workflow - that's difference.